packages/block-editor/src/filters/with-elements-visibility/controls.js

```diff
@@ -1,33 +1,16 @@
 import { __ } from "@wordpress/i18n";
 
-import { CardsManager, ControlsGroup, ControlsSection, ControlsTab, ToggleGroup } from "../../components";
+import { CollectionManager, ControlsGroup, ControlsSection, ControlsTab } from "../../components";
```

Comment:
It'd be better to use a less ambiguous name for this component like `ElementsVisibilityControls` or `ElementsVisibilityToggles`